go/vt/schemadiff/types.go

```diff
@@ -120,8 +120,8 @@ const (
 )
 
 const (
-	EnumReorderStrategyReject int = iota
-	EnumReorderStrategyAllow
+	EnumReorderStrategyAllow int = iota
```

Comment:
Subsequently to the main change in this PR, it's now fine to reorder enum values, and the default `schemadiff` hints enum-reordering strategy is now `EnumReorderStrategyAllow`